src/scss/components/jump-menu/_jump-menu.scss

```diff
@@ -0,0 +1,82 @@
+@use "../../tokens/color" as *;
+@use "../../tokens/spacing" as *;
+@use "../../tokens/screens" as *;
+
+.iati-jump-menu {
+  max-width: 30rem;
+  &__header {
+    background-color: $color-green-30;
+    padding: 0.5rem 1rem;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    @media (min-width: $screen-md) {
+      background: none;
+      padding: 0 0 0.5rem;
+    }
+  }
+  &__toggle {
+    background-color: transparent;
+    border: none;
+    font-weight: 500;
+    display: flex;
+    align-items: center;
+    gap: 0.5em;
+    &::after {
+      content: '';
+      display: block;
+      background-image: url("data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjQiIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggZD0iTTcuNzU3NDUgMTAuNTg1OEw5LjE3MTY2IDkuMTcxNTRMMTIuMDAwMSAxMkwxNC44Mjg1IDkuMTcxNTdMMTYuMjQyNyAxMC41ODU4TDEyLjAwMDEgMTQuODI4NEw3Ljc1NzQ1IDEwLjU4NThaIiBmaWxsPSJibGFjayIvPgo8cGF0aCBmaWxsLXJ1bGU9ImV2ZW5vZGQiIGNsaXAtcnVsZT0iZXZlbm9kZCIgZD0iTTEgMTJDMSA1LjkyNDg3IDUuOTI0ODcgMSAxMiAxQzE4LjA3NTEgMSAyMyA1LjkyNDg3IDIzIDEyQzIzIDE4LjA3NTEgMTguMDc1MSAyMyAxMiAyM0M1LjkyNDg3IDIzIDEgMTguMDc1MSAxIDEyWk0xMiAyMUM3LjAyOTQ0IDIxIDMgMTYuOTcwNiAzIDEyQzMgNy4wMjk0NCA3LjAyOTQ0IDMgMTIgM0MxNi45NzA2IDMgMjEgNy4wMjk0NCAyMSAxMkMyMSAxNi45NzA2IDE2Ljk3MDYgMjEgMTIgMjFaIiBmaWxsPSJibGFjayIvPgo8L3N2Zz4K");
```

Comment:
We've now configured vite to automatically inline SVGs into the compiled CSS, so we can keep the SVG files separate to improve readability. Please could you add the SVG file to the `src/assets/svg` folder and then reference it like in the [country-switcher](https://github.com/IATI/design-system/blob/main/src/scss/components/country-switcher/_country-switcher.scss#L26) component?